README.md

```diff
@@ -9,25 +9,48 @@ Datasets to train supervised classifiers for Named-Entity Recognition
  * leNER-br
  * Peres2017
  * [WikiANN](https://huggingface.co/datasets/wikiann)
+ * [WikiNEuRal](https://github.com/Babelscape/wikineural)
+
 
 <a name="de"></a>
 ### German
  * [GermEval2014](https://github.com/davidsbatista/NER-datasets/tree/master/GermEval2014)
  * [Europeana Newspapers](https://github.com/EuropeanaNewspapers/ner-corpora)
  * [WikiANN](https://huggingface.co/datasets/wikiann)
+ * [WikiNEuRal](https://github.com/Babelscape/wikineural)
  
 <a name="nl"></a>
 ### Dutch
  * [Europeana Newspapers](https://github.com/EuropeanaNewspapers/ner-corpora)
  * [WikiANN](https://huggingface.co/datasets/wikiann)
+ * [WikiNEuRal](https://github.com/Babelscape/wikineural)
 
 <a name="fr"></a>
 ### French
  * [Europeana Newspapers](https://github.com/EuropeanaNewspapers/ner-corpora)
  * [WikiANN](https://huggingface.co/datasets/wikiann)
+ * [WikiNEuRal](https://github.com/Babelscape/wikineural)
 
 <a name="en"></a>
 ### English
  * [CONLL2003](https://github.com/davidsbatista/NER-datasets/tree/master/CONLL2003)
  * [W-NUT2017](https://github.com/leondz/emerging_entities_17) Workshop on Noisy User-generated Text: Emerging and Rare entity recognition
  * [WikiANN](https://huggingface.co/datasets/wikiann)
+ * [WikiNEuRal](https://github.com/Babelscape/wikineural)
+
+<a name="it"></a>
```

Comment:
can you just give the same alignment here as for the other languages

Reply:
excuse me @davidsbatista, I didn't understand :( 
Could you kindly better explain which alignment are you referring to?
Thanks in advance

p.s. if you refer to the space before "### Language", it is present also for the new languages, but since there is the "+" symbol it seems that there is no space

Reply:
ah okie dokie